ipfsapi/encoding.py

```diff
@@ -128,6 +242,20 @@ class Pickle(Encoding):
     """
     name = 'pickle'
 
+    def __init__(self):
```

Comment:
Yes, something along the lines of "actually we're lying and all you data will be buffered by this". :wink: